wkb.go

```diff
@@ -77,6 +78,15 @@ func (p *Point) Scan(value interface{}) error {
 		return ErrUnsupportedDataType
 	}
 
+	//hex string
```

Comment:
This is basically workaround for a bug in https://github.com/go-pg/pg because it works just fine in https://github.com/lib/pq It seems like this is a workaround they are aware of, see https://github.com/go-pg/pg/blob/6480d2e890cfb57acde11306cb1f8a9748ea4d86/types/scan.go#L75

Converting binary data from postgres to a hex string and then making it a []byte type passed in here is completely unnecessary.

anyway, the comment here should be expanded to reflect this information.